### ISSUE TITLE
YourBase currently requires Java

### ISSUE THREAD

**Issue by nictuku** [yourbase/yourbase#7]
YourBase currently requires users to install Java because it is a requirement for [Bazel](https://bazel.build). 

Developers and security folks are not a fan of having to install Java (and keep it up to date) in order to build their Go, nodeJS or Ruby app.

We should look into ways of not requiring Java for YourBase. A few alternatives:

- don't use Bazel and find other tools that do the same job just as well;
- use the [Go skylark interpreter](https://github.com/google/skylark) to build a simple clone of Bazel. That is not a small project but it could make a lot of people happy.

**Comment by nictuku** [comment]
Some notes about a Go implementation here: https://groups.google.com/forum/#!topic/skylark-go/l07hJiMrFtM

You have to be a member of skylark-go to read this message, but anyone can join.

**Comment by kbfastcat** [comment]
yeah, this is something to consider, if bazel is the tool or not for the end consumer.

**Comment by philwo** [comment]
Bazel does not require Java to be installed. The default binaries that we officially provide on https://github.com/bazelbuild/bazel/releases bundle an OpenJDK, so it's completely self-contained and you can call it like any other binary.

We're going to ask Homebrew if they can use that version instead of adding a dependency on Java, due to user demand (as seen here). So, yes, technically Bazel requires Java, but we're working hard that no end user has to care about this. :)

Would this help you?

**Comment by nictuku** [comment]
@philwo making brew not need a separate Java would be helpful for sure!
